packages/eslint-config/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@loopback/eslint-config",
+  "version": "1.0.0-1",
+  "description": "Shared configuration for ESLint",
+  "engines": {
+    "node": ">=8.9"
+  },
+  "main": ".eslintrc.js",
```

Please remove the dot from the file name. Dot files are used for package infrastructure configuration. This file is not an infrastructure for `eslint-config`, it's an artifact exported by the package.

```suggestion
  "main": "eslintrc.js",
```


See also https://github.com/strongloop/eslint-config-loopback/blob/master/eslint.json and https://github.com/strongloop/eslint-config-loopback/blob/master/package.json#L5

```json5
{
  // ...
  "main": "eslint.json",
  // ...
}
```